docs/mediapipe.md

```diff
@@ -0,0 +1,191 @@
+
+## Configuration files <a name="configuration-files"></a>
+MediaPipe graph configuration is to be placed in the same json file like the 
+[models config file](starting_server.md).
+While models are defined in section `model_config_list`, graphs are configured in
+the `mediapipe_config_list` section. 
+
+Basic graph section template is depicted below:
+
+```
+
+{
+    "model_config_list": [...],
+    "mediapipe_config_list": [
+    {
+        "name":"mediaDummy",
+        "base_path":"/mediapipe/graphs/",
+        "graph_path":"graphdummyadapterfull.pbtxt",
+        "subconfig":"subconfig_dummy.json"
+    }
+    ]
+}
+
+```
+
+Basic subconfig:
+
+```
+
+{
+    "model_config_list": [
+        {"config": {
+                "name": "dummy",
+                "base_path": "/models/dummy",
+                "shape": "(1, 10)"
+        	}
+	}
+    ]
+}
+
+
+```
+Nodes in the MediaPipe graphs can reference both to the models configured in model_config_list section and in subconfig.
+
+### MediaPipe configuration options explained
+
+|Option|Type|Description|Required|
+|:---|:---|:---|:---|
+|`"name"`|string|Graph identifier related to name field specified in gRPC/REST request|Yes|
+|`"base_path"`|string|Path to the which graph definition and subconfig files paths are relative. May be absolute or relative to the main config path. Default value is "{main config path}\"|No|
+|`"graph_path"`|string|Path to the graph proto file. May be absolute or relative to the base_path. Default value is "{base_path}\graph.pbtxt". File have to exist.|No|
+|`"subconfig"`|string|Path to the subconfig file. May be absolute or relative to the base_path. Default value is "{base_path}\subconfig.json". Mssing  file does not result in error.|No|
+
+Subconfig file may only contain *model_config_list* section  - in the same format as in [models config file](starting_server.md).
+
+## Using Mediapipe <a name="using-mediapipe"></a>
+
+MediaPipe graphs can use the same KServe Inference API as the models. There are exactly the same calls for running
+the predictions. The request format must match the pipeline definition inputs.
+
+Graphs can be queried for their state using the calls [GetModelStatus](model_server_grpc_api_kfs.md)
+and [REST Model Status](model_server_rest_api_kfs.md)
+
+## MediaPipe Graps Examples <a name="graphs-examples"></a>
+
+[Image classification](../demos/mediapipe/image_classification/README.md)
+
+[Multi model](../demos/mediapipe/multi_model_graph/README.md)
+
+## Current limitations <a name="current-limitations"></a>
+
+- Making changes in subconfig file does not trigger config reloads. Main config changes are monitored and triggers subconfig reload even if those werent changed.
+
+- As it is preview version of the feature - MediaPipe graphs are supported only for GRPC KFS API. Only TFS calls supported are get model status and config reload.
```

Comment:
a preview status should be more visible with info that is it not ready to be used in production - it is not only related to TFS and REST.